app/src/types/event.ts

```diff
@@ -169,3 +169,20 @@ export const initialEvent: IEditEvent = {
   organizerEmail: parseEmail(''),
   maxParticipants: parseMaxAttendees(''),
 };
+
+export const maybeParseEvent = (eventContract: IEventContract): IEvent => {
+  const deserializedEvent = deserializeEvent(eventContract);
+  const domainEvent = parseEvent(deserializedEvent);
+  if (isOk(domainEvent)) {
+    return domainEvent.validValue;
+  }
+  // Man får egt bare lov å kaste new Error
+  // men det er tull
+  // eslint-disable-next-line
+  throw {
+    status: 'ERROR',
+    userMessage:
+      'Arrangementobjektet kan ikke parses av følgende grunner: ' +
+      domainEvent.errors.map(x => x.message).join(', '),
+  };
+};
```

Comment:
Er det vanlig å throwe ting som ikke er errors da?
Kan ikke dette 'come back and bite us' etterhvert?
La oss si noen bruker denne down the road og antar at den thrower en error så sprenges appen runtime

Reply:
Dette er ein interessant case. Det håndteres med ein typeguard, kor vi rethrower alt som ikkje matcher typen vår `INotification`. 
![image](https://user-images.githubusercontent.com/25415972/74637386-32453100-516a-11ea-97f8-2a511f6c69eb.png)

Det kan godt tenkes at denne typen burde vore ei klasse som arver frå `Error`, men da må vi jobbe med den klassen rundt om i appen i staden for å kunne opprette dette objektet som man ønsker.